### PR TITLE
Company Effectivness Indicators

### LINKED ISSUE #250: Company Effectivness Indicators
**Describe the solution you'd like**
Now that Torn shows the details behind a persons effectiveness, it would be great if TornTools could change the colour of the number *IF* the person has a negative value within their effectiveness details.

### PR DESCRIPTION
**Pull request by Sashank999**
Closes #250 .

### CODE REVIEW

File: torntools/scripts/content/company/ttCompany.js [hunk 1]
```diff
@@ -63,3 +65,13 @@ function showUserInfo() {
 			console.log("ERROR", err);
 		});
 }
+
+function changeEmployeesEffectivenessColor() {
+	requireElement("div.employee-list-wrap").then(() => {
+		for (let playerEffectiveness of doc.findAll("ul.employee-list div.effectiveness.clearfix")) {
+			if (playerEffectiveness.getAttribute("aria-label").split(":").map((x) => x.replace(/[a-zA-Z ;]/g, "")).map((x) => parseInt(x)).filter((x) => !isNaN(x)).some((x) => x < 0)) {
```
Comment (DeKleineKobini) [inline torntools/scripts/content/company/ttCompany.js:72]: This also triggers if you have a `-` in your name.

Adding `.slice(1)` after the `split` will throw away the name element from that array.

**Review by DeKleineKobini — CHANGES_REQUESTED** [review]
Don't forget the changelog.

File: torntools/scripts/content/company/ttCompany.js (hunk 1 above)
Comment (Sashank999) [reply torntools/scripts/content/company/ttCompany.js]: Fixed.

**Comment by Sashank999** [comment]
> Don't forget the changelog.

Done.

**Review by DeKleineKobini — APPROVED** [review]

**Pull request merged**